resources/aws/securitygroup.go

```diff
@@ -0,0 +1,89 @@
+package aws
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/giantswarm/clustertpr"
+	microerror "github.com/giantswarm/microkit/error"
+)
+
+const (
+	sshPort = 22
+)
+
+type SecurityGroup struct {
+	Description string
+	GroupName   string
+	VpcID       string
+	Cluster     clustertpr.Cluster
```

Comment:
Where do you want to create that list of ports, in which function and module, and based on what input?